commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/common/TsvPagetabExtension.kt

```diff
@@ -0,0 +1,7 @@
+package ac.uk.ebi.biostd.common
+
+class TsvPagetabExtension(
+    private val featureEnabled: Boolean = true
```

Comment:
I think by default it needs to be "pagetab.tsv" not the other way around. I mean the default behavior is the safest option (current behavior so system does not change.)